clients/web/src/views/widgets/ApiKeyListWidget.js

```diff
@@ -0,0 +1,169 @@
+girder.TokenScopeFull = {
```

Comment:
Thinking about it more, I think that would be a better way to do it, so that plugins that want to add scopes only have to add stuff on the server side.